impl/src/templates/crate.BUILD.template

```diff
@@ -4,41 +4,44 @@ cargo-raze crate build file.
 
 DO NOT EDIT! Replaced on runs of cargo-raze
 """
-package(default_visibility = [
-  # Public for visibility by "@raze__crate__version//" targets.
-  #
-  # Prefer access through "{{workspace.workspace_path}}", which limits external
-  # visibility to explicit Cargo.toml dependencies.
-  "//visibility:public",
-])
-
-licenses([
-  "{{crate.license.rating}}", # {{crate.license.name}}
-])
 
+# buildifier: disable=load
 load(
     "@io_bazel_rules_rust//rust:rust.bzl",
-    "rust_library",
     "rust_binary",
+    "rust_library",
     "rust_test",
 )
 
-{% set crate_name_sanitized = crate.pkg_name | replace(from="-", to="_") %}
+package(default_visibility = [
+    # Public for visibility by "@raze__crate__version//" targets.
+    #
+    # Prefer access through "{{workspace.workspace_path}}", which limits external
+    # visibility to explicit Cargo.toml dependencies.
+    "//visibility:public",
+])
+
+licenses([
+    "{{crate.license.rating}}",  # {{crate.license.name}}
+])
+
+# Generated targets
+{%- set crate_name_sanitized = crate.pkg_name | replace(from="-", to="_") %}
 {%- if crate.build_script_target %}
-{%- include "templates/partials/build_script.template" %}
+{%    include "templates/partials/build_script.template" %}
 {%- endif %}
 {%- for target in crate.targets %}
 {%- set target_name_sanitized = target.name | replace(from="-", to="_") %}
 {%- if target.kind == "lib" %}
-{%- include "templates/partials/rust_library.template" %}
+{%  include "templates/partials/rust_library.template" %}
```

Comment:
Here and below: also take the opportunity to fix the indentation?

Reply:
Like, just remove the spaces within the brackets? I still don't think I understand what the indentation is supposed to look like 😅

Reply:
The current logic is hard to follow, Actually there should be space before too since there is a for. So for this line that would be introducing 4 spaces:

```suggestion
{%      include "templates/partials/rust_library.template" %}
```

The idea is to be able to read the template in a way that is logical, for instance I completely missed the fact that all those `if`s  were included into a for loop because of missing indentation.

Reply:
Like so? da41d6ad671f3c94b330c2dc026df5fdb50bc462